Kotlin/demo/src/main/res/layout/activity_main.xml

```diff
@@ -1,43 +1,119 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
-    android:padding="24dp">
+    android:padding="24dp"
+    tools:ignore="rtlHardcoded">
 
     <EditText
         android:id="@+id/etInput"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_gravity="center_horizontal"
+        android:autofillHints="@null"
         android:gravity="center_horizontal"
         android:hint="@string/hint_blurhash"
         android:inputType="text"
-        android:singleLine="true"
+        android:maxLines="1"
         android:text="LEHV6nWB2yk8pyo0adR*.7kCMdnj"
-        android:textColor="@color/colorAccent" />
+        android:textColor="@color/textColor"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="HardcodedText" />
```

Comment:
How about moving the text into strings.xml with some name like `demo_string` to remove the `tools:ignore` here as well?

Reply:
Fixed